k8s/admin/monitoring/patches/cftptl/cluster-00/prom-operator.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: prom-operator
+  namespace: monitoring
+  annotations:
```

Comment:
See if you can move this to base template so that you don't need to add it to every patch

Reply:
Moved the annotations. The "name" and "namespaces" fields must be kept otherwise kustomize will throw an error.